pkgs/version/version_test.go

```diff
@@ -49,7 +49,7 @@ func TestGetVersionType(t *testing.T) {
 	// breaking test
 	commitMessage := "breaking - this is a new feature"
 
-	versionType := GetVersionType(commitMessage, commitTypes)
+	versionType := GetVersionType(commitMessage, commitTypes, "")
```

Comment:
Why empty string? This makes the test pass regardless of the function outcome. You need to pass the keyword that is expected

Reply:
vice versa! if you provide a keyword it will return it, so that the GetVersionType will not run till the end.
see here - https://github.com/online-applications/version-action/blob/b1e2e657bf002422e0524f326dc52b8e7fd96ea5/pkgs/version/version.go#L69